### ISSUE TITLE
Create base UI component

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#37]
## User story
1. As a user
2. I want a uniform interface throughout all the different views
3. So that I can easily switch between views and see/disconnect from the endpoint

## Acceptance criteria
* [x] The base component should show the currently connected endpoint
* [x] User can disconnect from current endpoint, as a result he should then be redirected to the landingpage
* [x] A basic navbar/viewselection which changes and indicates the current view (default shall be the cluster view)
=> Use the [mockup](https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/blob/main/Documentation/amos_mockup.pdf) inspiration

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
